website_wow/static/src/js/website_wow.js

```diff
@@ -0,0 +1,39 @@
+/* Copyright 2017 LasLabs Inc.
+   License GPL-3.0 or later (http://www.gnu.org/licenses/gpl).
+*/
+
+odoo.define('website_wow.website_wow', function (require) {
+    'use strict';
+
+    var base = require('web_editor.base');
+    var core = require('web.core');
+
+    var WebsiteWow = {
```

Comment:
Yeah, you can write an inline module, [just like it happened with the CSRF token in v9](https://github.com/odoo/odoo/blob/9.0/addons/web/views/webclient_templates.xml#L306-L310). Then your other module should depend on that one.